ignite/contrib/handlers/clearml_logger.py

```diff
@@ -577,18 +560,13 @@ class GradsHistHandler(BaseWeightsHistHandler):
             )
     """
 
-    def __init__(self, model: Module, tag: Optional[str] = None):
-        super(GradsHistHandler, self).__init__(model, tag=tag)
-
     def __call__(self, engine: Engine, logger: ClearMLLogger, event_name: Union[str, Events]) -> None:
         if not isinstance(logger, ClearMLLogger):
             raise RuntimeError("Handler 'GradsHistHandler' works only with ClearMLLogger")
 
         global_step = engine.state.get_event_attrib_value(event_name)
         tag_prefix = f"{self.tag}/" if self.tag else ""
-        for name, p in self.model.named_parameters():
-            if p.grad is None:
```

Comment:
Why do you remote this `if p.grad is None` ?

Reply:
I restored it.